src/utils/device.ts

```diff
@@ -83,11 +83,16 @@ export const canAccessDevice = (
 		)
 		.catchReturn(false);
 
+interface VpnHost {
+	id: string;
```

Comment:
This will actually be returned as a number I'm pretty sure
```suggestion
	id: number;
```

Reply:
Does that mean registering a service instance registration will return a number, not a string?